spec/requests/requests_requests_spec.rb

```diff
@@ -48,6 +48,43 @@
           expect(response).to have_http_status(:not_found)
         end
       end
+
+      context 'When organization has a default storage location' do
+        let(:request) { create(:request, organization: create(:organization, default_storage_location: 1)) }
+        it 'shows the column Default storage location inventory' do
+          get request_path(request)
+
+          expect(response.body).to include('Default storage location inventory')
+        end
+      end
+
+      context 'When organization does not have a default storage location' do
```

Comment:
I would combine the two does not have cases into one case so it is clear that you are testing for the case where neither the organization nor the partner has a default storage location.